.eslintrc

```diff
@@ -93,6 +93,14 @@
             "node": false,
             "jest": false
         }
+    }, {
+        "files": ["website/**/*.js"],
+        "extends": [
+            "plugin:react/recommended"
+        ],
+        "rules": {
+            "react/prop-types": 0
+        }
```

Comment:
ESLint has issues with shared configs and PnP (https://github.com/yarnpkg/berry/issues/8), so I've moved `website/.eslintrc` to here.